src/parser/metrics/Classes.ts

```diff
@@ -58,7 +90,14 @@ export class Classes implements Metric {
             metricValue: matches.length,
         };
     }
-
+    addQueriesForJS_TS_TSX() {
+        this.statementsSuperSet.push(
+            new SimpleLanguageSpecificQueryStatement(
+                "(type_alias_declaration value: (object_type))",
+                ["ts", "tsx"],
+            ),
+        );
+    }
```

Comment:
Please place the function directly below or above the similar `addQueriesForCAndCpp` function so that all special handling can be found easily.